daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1787,8 +1762,9 @@ private[lf] object SBuiltin {
     }
   }
 
-  /** $error :: Text -> a */
-  final case object SBError extends SBuiltin(1) {
+  /** $userError :: Text -> Error */
+  final case object SBUserError extends SBuiltin(1) {
```

Comment:
Renamed to be consistent with other builtins that return `Control.Error` values

Reply:
It was consistent with the LF Builtin `BError`.
Maybe we need a convention to distinguish builtins that corresponds to LF Builtins and those that does not.

Reply:
Absolutely agree. Not for this PR, but I'm thinking that at some point in time we need to start grouping Speedy builtin's in some (formal) manner (c.f. pure and on-ledger builtins). At that point in time, we should be able to manage such concerns.

Reply:
fair enough.